mlir/test/Dialect/Vector/vector-warp-distribute.mlir

```diff
@@ -620,6 +620,38 @@ func.func @vector_reduction(%laneid: index) -> (f32) {
 
 // -----
 
+// CHECK-PROP-LABEL: func @warp_distribute(
+// CHECK-PROP-SAME:   %[[ID:.*]]: index, %[[SRC:.+]]: memref<128xf32>, %[[DEST:.+]]: memref<128xf32>)
+// CHECK-PROP:   vector.warp_execute_on_lane_0(%[[ID]])[32]
+// CHECK-PROP-NEXT:     "some_def"() : () -> vector<4096xf32>
+// CHECK-PROP-NEXT: %{{.*}} = vector.reduction
+// CHECK-PROP-DAG: %[[DEF:.*]] = arith.divf %{{.*}}, %{{.*}} : vector<1xf32>
```

Comment:
The `-DAG` is unnecessary here because there's only one line to match (instead of a DAG)